integ/components/deadline/common/scripts/bash/component_e2e.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+COMPONENT_NAME=${1:-undefined}
+OPTION=${2:-undefined}
+
+if [[ $(basename $(pwd)) != $COMPONENT_NAME ]]; then
+  echo "ERROR: Script must be run from top directory of test component"
+  exit 1
+fi
+
+SKIP_TEST_CHECK=\$SKIP_${COMPONENT_NAME}_TEST
+SKIP_TEST_CHECK=$(eval "echo $SKIP_TEST_CHECK" 2> /dev/null) || SKIP_TEST_CHECK=false
+if [[ ! "${SKIP_TEST_CHECK}" == "true" ]]; then
```

Comment:
I guess `==` and `=` are equal in `bash`, but in other `POSIX` shells only `=` is guaranteed to work, so I think we should go with a safer `=` option. Please correct me if I'm wrong.

There are multiple places in this PR where this could be changed.

Reply:
I wasn't aware of that but as far as I know `=` should work just as well; I'll make a quick pass to fix these.